wandb/sdk/wandb_artifacts.py

```diff
@@ -794,6 +794,9 @@ def versioning_enabled(self, bucket):
         return self._versioning_enabled
 
     def load_path(self, artifact, manifest_entry, local=False):
+        if not local:
```

Comment:
I think we also want to update the `WBArtifactHandler` handler's `load_path` such that it does something like this:

```
    def load_path(self, artifact, manifest_entry, local=False):
        # Parse the reference path and download the artifact if needed
        artifact_id = util.host_from_path(manifest_entry.ref)
        artifact_file_path = util.uri_from_path(manifest_entry.ref)

        dep_artifact = PublicArtifact.from_id(
            util.hex_to_b64_id(artifact_id), self.client
        )
        
        if local:
            link_target_path = dep_artifact.get_path(artifact_file_path).download()
        else:
            link_target_path = dep_artifact.get_path(artifact_file_path).ref()

        return link_target_path
```